keyboards/boardsource/lulu/keymaps/rmeli/keymap.c

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022 Cole Smith <cole@boadsource.xyz>
+Copyright 2022 Rocco Meli <@RMeli>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#include "rmeli.h"
+
+enum layers { _QWERTY, _COLEMAK_DH, _RAISE, _LOWER, _ADJUST };
```

Comment:
```suggestion
enum layers { 
    _QWERTY, 
    _COLEMAK_DH, 
    _RAISE, 
    _LOWER, 
    _ADJUST, 
};
```

Reply:
This is how it was automatically formatted by `clang-format`:

```bash
clang-format -i --style=file:.clang-format users/rmeli/km_lulu/keymap.c
```

Do I need to change it and turn off `clang-format` for this `enum`?

Reply:
Note the extra comma after `_ADJUST`?   It prevents it from being formatted weird.  Just a weird ... behavior, in general.

Reply:
I missed the comma. Fixed!